go.mod

```diff
@@ -91,6 +93,8 @@ require (
 
 replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.14.1-0.20220621123826-1e21b88b991f
 
-replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.16.1-0.20220621124242-4fe813879da6
+replace github.com/jfrog/jfrog-cli-core/v2 => ../jfrog-cli-core
```

Comment:
Please replace it by a URL to your branch of jfrog-cli-core, for example:
```
 replace github.com/jfrog/jfrog-cli-core/v2 => github.com/omerzi/jfrog-cli-core/v2 <hash>
```
This will help us to see the tests compiling and running.

Reply:
done